### DISCUSSION TITLE
Seasonal themes and intro sequences

### DISCUSSION THREAD

**Discussion by ItsShamed** [ppy/osu#24403] [Ideas] [answered]
> **Note**
> This is not the same thing as #15169 and #15722

Currently on special events (solstices, Halloween etc.), stable gets a  custom menu theme song and sometimes a decorated osu! cookie along with the usual seasonal backgrounds.
It would be cool to have that back on lazer along with custom intros sequences as well

**Comment by peppy** [comment]
it would be cool

**Reply by ItsShamed** [reply, marked as answer]
#31206 :)

**Comment by LukynkaCZE** [comment]
If I wanted to implement this, how should I got about this? Would it be another type of intro in the "Intro Sequence" setting that would be force set during special events (Halloween, Winter Holidays etc.)?

**Reply by LukynkaCZE** [reply]
In addition, should the intro sequence be selectable throughout the entire year in the dropdown or just during that special event

**Reply by bdach** [reply]
You probably shouldn't try to implement this because there are concerns of automated deployment alongside stable which are likely going to require internal tooling.

**Reply by LukynkaCZE** [reply]
okay!
